examples/experimental/tutorial_dbt_dagster_v2/tutorial_dbt_dagster_v2/assets/dependencies.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+
+import pandas as pd
+from dagster import AssetIn, AssetOut, OpExecutionContext, asset, multi_asset
+from dagster_dbt.asset_decorator import dbt_assets
+from dagster_dbt.cli import DbtCli, DbtManifest
+
+from ..constants import MANIFEST_PATH
+
+manifest = DbtManifest.read(path=MANIFEST_PATH)
+
+
+@multi_asset(
+    outs={
+        name: AssetOut(key=asset_key)
+        for name, asset_key in manifest.get_asset_keys_by_output_name_for_source("raw_data").items()
+    },
+)
+def raw_inventory() -> Any:
+    return pd.DataFrame()
```

Comment:
Can we handle the case where:
- A dbt `source_name` just consists of one `table_name`, and
- The user uses `@asset` instead of using `@multi_asset`?